data/human/free worlds weapons.txt

```diff
@@ -107,4 +107,4 @@ outfit "Plasma Repeater Turret"
 		"hull damage" 13
 		"heat damage" 144
 		"hit force" 19
-	description "This turret places two Plasma Repeaters on one big mount, with a creative twist: The firing clocks have been tweaked so each individual gun fires slightly faster, allowing the turret to overall more effectively saturate an area. Because of its effectiveness, the Free Worlds have restricted the sale of this weapon to its Militia."
+	description "This turret places two Plasma Repeaters on one big mount, with a creative twist: The firing clocks have been tweaked so each individual gun fires slightly faster, allowing the turret to overall more effectively saturate an area. Because of its effectiveness, the Free Worlds have restricted the sale of this weapon to the Militia alone."
```

Comment:
I think the previous way it's written matches the Skein description? I might be wrong.

Reply:
Only was after it in the Skein description, but you're right.
```suggestion
	description "This turret places two Plasma Repeaters on one big mount, with a creative twist: The firing clocks have been tweaked so each individual gun fires slightly faster, allowing the turret to overall more effectively saturate an area. Because of its effectiveness, the Free Worlds have restricted the sale of this weapon to its Militia only."
```

Reply:
I think "its Militia" is the best phrasing. It's concise.